room/room-compiler-processing/src/main/java/androidx/room/compiler/processing/javac/JavacBasicAnnotationProcessor.kt

```diff
@@ -31,43 +34,58 @@ import javax.lang.model.element.Element
 abstract class JavacBasicAnnotationProcessor :
```

Comment:
You should also update `KspBasicAnnotationProcessor` to not call a `xStep` if there are no annotated elements for it too.

Reply:
Done.